test/e2e/app-dir/actions/app-action.test.ts

```diff
@@ -1256,7 +1256,8 @@ describe('app-dir action handling', () => {
         expect(await browser.url()).toBe(`${next.url}/pages-dir`)
         expect(mpaTriggered).toBe(true)
       })
-    })
+      // TODO: remove custom duration in case we increase the default.
```

Comment:
You can still keep default values. This helps readability in cases where the timeout is meaningful. The default wouldn't be since `retry` is generic whereas this timeout is tied to compilation speed. 

And a TODO for "in case" seems also inappropriate
```suggestion
```